pkg/config/config.go

```diff
@@ -35,12 +34,12 @@ const (
 	flagResourceTags         = "resource-tags"
 	flagWatchNamespace       = "watch-namespace"
 	flagEnableWebhookServer  = "enable-webhook-server"
-	flagWebhookServerAddr    = "webhook-server-addr"
+	flagWebhookServerPort    = "webhook-server-port"
+	flagWebhookServerHost    = "webhook-server-host"
```

Comment:
I know this is not consistent with the all the other flag (`--metrics-addr` for example). Shall i use a single flag instead ? keeping the `--webhook-server-addr` and parse/split the host/port ?

Reply:
I recommend keeping the consistency with "metrics-addr".

IMO, separating port and host provides good user experience but ACK customers will not be dealing with these directly, correct? They will just install using helm charts. So I am fine with keeping it as "Addr".

Reply:
Yep agreed :ok_hand:  I kept `webhook-server-addr` and just deleted `--bind-port` flag. PTAL